modules/phase_field/src/action/GrandPotentialKernelAction.C

```diff
@@ -248,7 +259,7 @@ GrandPotentialKernelAction::act()
     InputParameters params = _factory.getValidParams("SusceptibilityTimeDerivative");
     params.set<NonlinearVariableName>("variable") = w_names[i];
     params.set<MaterialPropertyName>("f_name") = chis[i];
-    params.set<std::vector<VariableName>>("args") = v1;
+    params.set<std::vector<VariableName>>("args") = v0;
```

Comment:
why this change?

Reply:
v0 contains chemical potentials as well as order parameters. This change allows it to couple chemical potential variables.